lib/Core/Executor.cpp

```diff
@@ -4026,9 +4206,9 @@ void Executor::runFunctionAsMain(Function *f,
     }
   }
 
-  ExecutionState *state = new ExecutionState(kmodule->functionMap[f]);
+  ExecutionState *state = new ExecutionState(this, kmodule->functionMap[f], modOpts.EnableNvmInfo);
   // (iangneal): I want to start my stack frame with the main function
-  if (modOpts.EnableNvmInfo) {
+  if (false && modOpts.EnableNvmInfo) {
```

Comment:
What's the point of this? Just permanently disabled for now?

Reply:
Because you found my other secret debug thing that I forgot to remove. This is defunct and should be removed.